sagemaker/README.md

```diff
@@ -225,274 +225,3 @@ python3 launch/launch_gconstruct \
 
 For a full list of `Processor` parameters see:
 https://sagemaker.readthedocs.io/en/stable/api/training/processing.html
```

Comment:
Do we have the deleted information in the doc?

Reply:
Yes in the sense that we no longer need this guide because we only use SageMaker local mode. There's no way to generate Docker compose files any more. Everything is handled by the SM SDK.